PYTHON/utils/mecademic_state/mecademic_calculations.py

```diff
@@ -0,0 +1,103 @@
+"""
+This command limits the angular velocities of the robot joints. It affects the movements
+generated by the MovePose and MoveJoints commands.
+Arguments:
+- p: percentage of maximum joint velocities, ranging from 1 to 100%
+Default values:
+By default, the limit is set to 25%.
+Note that it is not possible to limit the velocity of only one joint. With this command, the
+maximum velocities of all joints are limited proportionally. In other words, the maximum
+velocity of each joint will be reduced to a percentage p of its top velocity. The top velocity
+of joints 1 and 2 is 150◦/s, of joint 3 is 180◦/s, of joints 4 and 5 is 300◦/s, and of joint 6 is
+500◦/s
+
+
+Parameters
+    - current: the joints present position (6 element array of numbers)
+    - next: the next position the joints will go to
+    - timeDelta: the amount of time (in milliseconds) that the move should take
+"""
```

Comment:
should these explanations not be moved into the docstrings for functions?

Reply:
Sorry about that, on it